README.md

```diff
@@ -54,7 +54,7 @@ Nothing changed [from before](https://github.com/CodeFX-org/demo-jigsaw-advent-c
 
 ### _chocolate_
 
-As [before with _factories_](https://github.com/CodeFX-org/demo-jigsaw-advent-calendar/tree/02-splitting-into-modules#factories) this module must `require public` the _surprise_ module.
+As [before with _factories_](https://github.com/CodeFX-org/demo-jigsaw-advent-calendar/tree/02-splitting-into-modules#factories) this module must `require transitive` the _surprise_ module.
```

Comment:
I assumed the "require" to be intentional, for the sake of grammatical correctness ("the module must require..."). But now that I think about it I agree that this might be confusing... maybe it would be better not to include "require" in backticks?